test/e2e/tests/request-queuing/dapp1-switch-dapp2-send.spec.js

```diff
@@ -119,19 +113,6 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
         // so we leave this delay until the issue is fixed (#27360)
         await driver.delay(5000);
 
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-
-        // Check correct network on the send confirmation.
-        await driver.findElement({
-          css: '[data-testid="network-display"]',
-          text: 'Localhost 8546',
-        });
-
-        await driver.clickElementAndWaitForWindowToClose({
-          text: 'Confirm',
-          tag: 'button',
-        });
-
```

Comment:
this should also still be valid, but I would assume the confirmation is on Localhost 8545

Reply:
Definitely, updated here https://github.com/MetaMask/metamask-extension/pull/27980/commits/7f7646f7a81ee1c998cfddde8a2185b9099fa80c